tools/gethwrappers/CallProxy.go

```diff
@@ -5,6 +5,7 @@ package gethwrappers
 
 import (
 	"errors"
+	"fmt"
```

Comment:
why move these to tools? I would assume that the generate files continue to live in gethwrappers (or in generate) and the tool to do so lives in tools?